aicsimageio/readers/tiff_glob_reader.py

```diff
@@ -614,3 +629,25 @@ def MicroManagerIndexer(path_to_img: Union[str, Path]) -> pd.Series:
         inds = re.findall(r"\d+", Path(path_to_img).name)
         series = pd.Series(inds, index=["C", "S", "T", "Z"]).astype(int)
         return series
+
+    @staticmethod
+    def get_channels_from_DisplaySettings(
+        display_settings_file: Union[str, Path]
+    ) -> Optional[List[str]]:
+        """
+        Extract the Channel Names from the MicroManager generated DisplaySettings.json file. This file
+        is generated when saving single tiff files and is the only way this metadata is accessible with that
+        MM 1.4 style saving method.
+        (TODO check that that statement is true. I inferred it from reading
+        https://github.com/micro-manager/micro-manager/blob/v2.0.0/mmstudio/src/main/java/org/micromanager/data/internal/StorageSinglePlaneTiffSeries.java # noqa
+        )
+        """
```

Comment:
I thinkkkkk this is true, but maybe @nicost would know if there is a better way to extract channel names?

This is for when you click "Separate Image Files" in the MDA dialog


![image](https://user-images.githubusercontent.com/10111092/140623565-681fe58c-023d-4ce0-8214-6625679e0880.png)

Also, am I correct that using that option is essentially using 1.4 style saving? We use it because we got burned a few times with jumbled order of files when using the stack,  but I'm thinking that maybe with better loading software (e.g. aics) that won't be so much of an issue.

Reply:
wow don't know how I missed this before.... there is a `metadata.txt` file

https://micro-manager.org/Micro-Manager_File_Formats